### ISSUE TITLE
Aggregate bonded sent from cosigner from multilevel multi-sig present as Unknown

### ISSUE THREAD

**Issue by cryptoBeliever** [symbol/desktop-wallet#1848]
Steps:
1. Create multilevel multi-sig. For example:
```
Multilevel (2 of 2)
 	Multisig cosigner 1 (1 of 1)
 		Cosigner 1
 	Multisig cosigner 2 (1 of 1)
 		Cosigner 2
```
2. Send transaction from Cosigner 1. 
3. Go to Cosigner 2 and open transaction details.

Result:
There is a view like for Unknown signer but this signer is from a Known cosigner. 

Expected result:
We should not show an "unknown signer warning" and present flow as for a known accounts.

**Comment by yilmazbahadir** [comment]
I think we can use this method to determine if an address is in the current multi-sig tree:

https://github.com/symbol/desktop-wallet/blob/8b236aa7191776709a42c1c7bff50ff0a27c832f/src/services/MultisigService.ts#L180

**Comment by OlegMakarenko** [comment]
@yilmazbahadir thanks mate!

**Comment by cryptoBeliever** [comment]
@OlegMakarenko we are using currently this method: https://discord.com/channels/@me/877119674742087720/945263935664324648

The issue is related to https://github.com/symbol/desktop-wallet/issues/1849. When we call multisig/graph for cosigner not the entire tree is returned. We can use proposed fix here https://github.com/symbol/desktop-wallet/issues/1849 (additional call) to fetch entire tree.

**Comment by cryptoBeliever** [comment]
Fixed in https://github.com/symbol/desktop-wallet/pull/1847

**Comment by cryptoBeliever** [comment]
Released in 1.0.11
